### PR TITLE
updated 3d slicer

### PR DESCRIPTION
**Pull request by raintwoto**
I have implemented the ft_read_sens file and ft_filetype, however, I was stucked at 'ft_convert_units' in the 'ft_write_sens' since I did not understand the units there and why it must be mm in other types.

### CODE REVIEW

File: fileio/private/read_nihonkohden_hdr.m [hunk 1]
```diff
@@ -0,0 +1,105 @@
+function [hdr] = read_nihonkohden_hdr(filename)
```
Comment (schoffelen) [inline fileio/private/read_nihonkohden_hdr.m]: Why is this function part of the PR? Is this function referenced in ft_read_header and or ft_read_data?

**Comment by schoffelen** [comment]
Why do you need to update ft_write_sens? Also, if the units are unknown, then I guess that they should remain unknown. ft_convert_units sometimes tries to do an educated guess as to what the units are. Yet, this does not depend on the file type / format.

File: fileio/private/read_nihonkohden_hdr.m (hunk 1 above)
Comment (raintwoto) [reply fileio/private/read_nihonkohden_hdr.m]: Sorry, that is from my last PR.....

**Comment by raintwoto** [comment]
> Why do you need to update ft_write_sens? Also, if the units are unknown, then I guess that they should remain unknown. ft_convert_units sometimes tries to do an educated guess as to what the units are. Yet, this does not depend on the file type / format.

I think you are right. I don't think we need the ft_write_sens too.

**Comment by schoffelen** [comment]
OK, thanks I will merge. Do you happen to have an example file that can be used for testing?

**Pull request merged**